http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterp.scala

```diff
@@ -89,15 +93,16 @@ class DiskServiceInterp[F[_]: Parallel](config: PersistentDiskConfig,
                             config,
                             req,
                             ctx.now,
-                            sourceDiskOpt
+                            sourceDiskOpt,
+                            parentWorkspaceId
               )
             )
             _ <- authProvider
               .notifyResourceCreated(samResource, userInfo.userEmail, googleProject)
               .handleErrorWith { t =>
                 log.error(t)(
                   s"[${ctx.traceId}] Failed to notify the AuthProvider for creation of persistent disk ${disk.projectNameString}"
-                ) >> F.raiseError(t)
+                ) >> F.raiseError[Unit](t)
```

Comment:
I don't know if it's just me, but my Intellij flags this as a compile error -- adding the `Unit` type param fixed it.

Reply:
Oh interesting, my IntelliJ never yelled at me for that, but it can't hurt, thanks for fixing :)